### PR TITLE
Error styles

### PR DESCRIPTION
**Pull request by tanojaja**

### CODE REVIEW

File: src/js/controllers/index.js [hunk 1]
```diff
@@ -1296,7 +1297,7 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     if (!self.usePushNotifications) return cb();
 
     pushNotificationsService.unsubscribe(walletId, function(err) {
-      if (err) $log.warn('Subscription error: ' + err.code);
+      if (err) $log.warn('Subscription error: ' + err.message);
```
Comment (gabrielbazan7) [inline src/js/controllers/index.js]: Unsubscription error

File: src/js/services/pushNotificationsService.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ angular.module('copayApp.services')
 
       lodash.forEach(profileService.getWallets(null), function(wallet) {
         root.unsubscribe(wallet.id, function(err) {
-          if (err) $log.warn('Subscription error: ' + err.code);
+          if (err) $log.warn('Subscription error: ' + err.message);
```
Comment (gabrielbazan7) [inline src/js/services/pushNotificationsService.js]: Unsubscription error

File: src/js/controllers/index.js (hunk 1 above)
Comment (tanojaja) [inline src/js/controllers/index.js]: Done. thks

File: src/js/services/pushNotificationsService.js (hunk 2 above)
Comment (tanojaja) [inline src/js/services/pushNotificationsService.js]: Done. thks

File: src/js/controllers/walletHome.js [hunk 3]
```diff
@@ -286,8 +286,7 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
         $rootScope.$on(eventName, function() {
           fc.getTx($scope.tx.id, function(err, tx) {
             if (err) {
-
-              if (err.code && err.code == 'TX_NOT_FOUND' &&
+              if (err.name && err.name == 'TX_NOT_FOUND' &&
```
Comment (gabrielbazan7) [inline src/js/controllers/walletHome.js]: err.message

File: src/js/services/bwsError.js [hunk 4]
```diff
@@ -4,50 +4,76 @@ angular.module('copayApp.services')
     var root = {};
 
     root.msg = function(err, prefix) {
+      var name;
+
+      if (err.name == 'Error')
```
Comment (cmgustavo) [inline src/js/services/bwsError.js]: What happen if `err` doesn't exist?

File: src/js/services/bwsError.js [hunk 5]
```diff
@@ -4,50 +4,76 @@ angular.module('copayApp.services')
...
+        name = err.name.replace(/^bwc.Error/g, '');
+      else
+        name = err;
+
       var body = '';
       prefix = prefix || '';
 
-      if (err && err.code) {
-        switch (err.code) {
+      if (err && name) {
```
Comment (cmgustavo) [inline src/js/services/bwsError.js]: If you already did check that `err` exists, the variable `name` is enough.

**Comment by cmgustavo** [comment]
Need rebase.

**Comment by tanojaja** [comment]
Updated

**Comment by cmgustavo** [comment]
ACK

**Pull request merged**